packages/cli/src/databases/entities/User.ts

```diff
@@ -113,4 +113,14 @@ export class User extends AbstractEntity implements IUser {
 	computeIsPending(): void {
 		this.isPending = this.password === null;
 	}
+
+	/**
+	 * Whether the user is instance owner
+	 */
+	isOwner: boolean;
+
+	@AfterLoad()
+	computeIsOwner(): void {
+		this.isOwner = this.globalRole?.name === 'owner';
+	}
```

Comment:
In many spots we're doing this check `!user?.isOwner` to target members, which requires the reader to know only two roles exist and requires for only owner and member roles to exist - any new non-owner role will also be caught by the checks. And RBAC with more roles is coming soon!

So maybe we could have a field for the actual role name and keep `isOwner` as a derivative of that new one, for readability and future-proofing? But if that requires too much refactoring, then let's drop it, just a thought.

Reply:
I'll keep it for now since it works and IMO we should do this refactor as part of the RBAC implementation